tests/Integration/bootstrap.php

```diff
@@ -45,7 +45,8 @@
 		exit( 1 );
 	}
 
-	// Include the Parsely custom test cases.
+	// Additional necessary requires.
+	require_once dirname( __DIR__ ) . '/trait-tests-trait.php';
 	require_once __DIR__ . '/TestCase.php';
 	require_once __DIR__ . '/StructuredData/NonPostTestCase.php';
```

Comment:
Why we're surrounding this with `dirname` and not the other requires?

Reply:
Because the trait's php file is in the parent directory, while the other two aren't.